src/index.html

```diff
@@ -1,13 +1,431 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
+    <title>Air</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&family=Poppins:wght@600;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon"
+    type="image/x-icon" href="/images/air-logo.svg">
   </head>
   <body>
-    <h1>Dia</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header header__container">
+      <div class="header__top">
+        <div class="header__content">
+          <a href="#" class="header__logo-link">
+            <img
+            src="./images/air-logo.svg"
+            alt="Air-logo"
+            class="header__logo"
+            >
+          </a>
+
+          <a href="#menu" class="nav__burger-menu"></a>
+
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#about-us">About us</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#services">Services</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#testimonials">Testimonials</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#contact-us">Contact us</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+
+        <a href="#send-us-a-message" class="header__button-link">
+          <button class="header__button">Hire us</button>
+        </a>
+      </div>
+
+      <section class="strategic page__strategic">
+        <h1 class="strategic__title">Strategic Agency</h1>
+        <h2 class="strategic__subtitle">We believe in the power of bold ideas that can solve business challenges.</h2>
+
+        <a href="#our-expertise" class="strategic__button">
+          <button class="strategic__button">Learn more</button>
+        </a>
+      </section>
+
+      <section class="slider slider__container">
+        <article class="slider__photo">
+          <div class="slider__items">
+            <div class="slider__arrow-container">
+              <span class="slider__arrow-left"></span>
+              <span class="slider__arrow-right"></span>
+            </div>
+            <div class="slider__bottom">
+              <p class="slider__title">Intro</p>
+              <p class="slider__subtitle">By the same illusion which lifts the horizon.</p>
+            </div>
+          </div>
+        </article>
+      </section>
+    </header>
+
+    <aside class="menu page__menu" id="menu">
```

Comment:
That whole HTML should be moved above right after `<a href="#menu" class="nav__burger-menu"></a>`

think you should remove class naming here to be something like e.g. `nav-mobile` and follow this name all rest above names, it will be more decriptive